sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosContainerProactiveInitConfig.java

```diff
@@ -19,14 +21,20 @@ public final class CosmosContainerProactiveInitConfig {
             .CosmosContainerIdentityHelper
             .getCosmosContainerIdentityAccessor();
     private final List<CosmosContainerIdentity> cosmosContainerIdentities;
+    private final Map<String, Integer> containerLinkToMinConnectionsMap;
```

Comment:
CosmosContainerIdentity is a public contract, I wonder whether mapping to an inner model will be easier. CosmosContainerIdentity -> CosmosContainerConfig (which includes all the properties from cosmosContainerIdentity + the minPool size + any other future properties)

Reply:
Great idea :)

Reply:
Let's iterate on this. I'll create an issue to track this.